src/idd.ml

```diff
@@ -190,6 +192,18 @@ let star mgr (d0:t) =
 let subseteq mgr (d0:t) (d1:t) =
   equal (union mgr d0 d1) d1
 
+
+
+let idd_from_dd mgr (dd:Dd.t) =
```

Comment:
1. `idd_from_dd` -> 'of_dd' (This is a standard OCaml convention. Outside the module, you will then write `Idd.of_dd`)
2. I would expect such a function to be semantics preserving, but as mentioned above DDs are uninterpreted diagrams without semantics. I think `bdd_from_dd` would make more sense.

I suggest we remove this for now?

Reply:
EDIT: I just saw your using this in the compiler. Here is my suggestion:
```
let of_bdd (bdd : Bdd.t) : t = (bdd :> Dd.t)
```
Then, in the documentation, say that the function is undefinded if the bdd contains output variables, but semantics preserving otherwise: `Bdd.eval bdd ~env = Idd.(eval (of_bdd bdd) ~env)`